app/src/test/java/io/crate/node/NodeSettingsTest.java

```diff
@@ -143,9 +144,11 @@ public void testCustomYMLSettings() throws IOException {
         File custom = new File("custom");
         custom.mkdir();
         File file = new File(custom, "custom.yml");
-        FileWriter customWriter = new FileWriter(file, false);
-        customWriter.write("cluster.name: custom");
-        customWriter.close();
+        try (FileWriter customWriter = new FileWriter(file, false)) {
+            customWriter.write("cluster.name: custom");
+        } catch (IOException e) {
+            fail(e.getMessage());
```

Comment:
also take a look at the other tests. I don't think we need to catch exceptions there just to fail